### DISCUSSION TITLE
How do I make the Social Login Icon bigger?

### DISCUSSION THREAD

**Discussion by DastardlyDino** [goauthentik/authentik#9813] [Q&A]
How can I customize this home page? Most importantly I would like to make the Plex icon MUCH bigger so it is more noticeable. I know that will be the preferred way for my friends and family to login. But I would also like to customize this whole "Welcome Page" if possible.

![image](https://github.com/goauthentik/authentik/assets/47339617/3cab52c3-433d-427c-a12f-436d43c70c3c)

**Comment by thaynes43** [comment]
I have the same question. On a phone in darkmode the Plex button is hard to see.

**Reply by DastardlyDino** [reply]
Still haven't figured this out. It's the main reason I'm still not using Authentik. I wish the community had more engagement with homelab users like myself.
